### ISSUE TITLE
Runtime tick on genesis block / code

### ISSUE THREAD

**Issue by steinerkelvin** [Kindelia/Kindelia#186]
Basically this should work:

```patch
diff --git a/src/hvm.rs b/src/hvm.rs
index 5a1aefa..3a58af7 100644
--- a/src/hvm.rs
+++ b/src/hvm.rs
@@ -1569,7 +1569,7 @@ pub fn init_runtime(heaps_path: PathBuf) -> Runtime {
   // TODO: add Genesis statements to actual block 0
   rt.run_statements_from_code(GENESIS_CODE, true, false);
 
-  rt.snapshot();
+  rt.tick();
   return rt;
 }
 ```
 
 And code following the genesis code should run on ticks 1+

**Comment by steinerkelvin** [comment]
fixed by 2eb9830

**Comment by steinerkelvin** [comment]
fixed by https://github.com/Kindelia/Kindelia/commit/2eb98303b17d427dfa591cfcdd039cabd0257a7c
